apps/meteor/ee/app/license/server/settings.ts

```diff
@@ -20,4 +20,11 @@ await settingsRegistry.addGroup('Enterprise', async function () {
 			i18nLabel: 'Status',
 		});
 	});
+	await this.add('Cloud_Workspace_AirGapped_Restrictions_Remaining_Days', -1, {
+		type: 'int',
+		hidden: true,
```

Comment:
Hidden settings are only available in the server level (means they cannot be fetched by the client or through any endpoint)

In order for the frontend to be able tu use this we have to remove this property. We can find a way to hide it in the admin panel if that's necessary.